BackgammonPlay.py

```diff
@@ -0,0 +1,256 @@
+import random
+
+
+class InvalidMoveError(Exception):
+    pass
+
+
+class Board(object):
+    def __init__(self):
+        self.board = [[] for i in range(24)]
+
+    def get_board(self):
+        return self.board
+
+    def start_board(self, sign1, sign2):
+        self.board[0] = 2 * [sign1]
+        self.board[5] = 5 * [sign2]
+        self.board[7] = 3 * [sign2]
+        self.board[11] = 5 * [sign1]
+        self.board[12] = 5 * [sign2]
+        self.board[16] = 3 * [sign1]
+        self.board[18] = 5 * [sign1]
+        self.board[23] = 2 * [sign2]
+
+    def change_board(self, index_now, steps, sign):
+        try:
+            if (sign in self.board[index_now] and (
+                    self.board[index_now + steps] == [] or sign in (self.board[index_now + steps]))):
+                self.board[index_now] = self.board[index_now][:-1]
+                self.board[index_now + steps].append(sign)
```

Comment:
this statement is complex and hard to read, split any check here to a diffrent function.